content/authentication/troubleshooting-ssh/using-ssh-over-the-https-port.md

```diff
@@ -28,6 +34,12 @@ $ ssh -T -p 443 git@ssh.github.com
 
 If that worked, great! If not, you may need to [follow our troubleshooting guide](/articles/error-permission-denied-publickey).
 
+## Git Client Repository Example
+
+```
+>git clone ssh://git@ssh.{% data variables.command_line.codeblock %}:443/YOUR-USERNAME/YOUR-REPOSITORY.git
```

Comment:
Formatting nit: we use `$ ` (dollar sign followed by space) as the prompt when we expect a user to type a command.

Reply:
Thank you! ✨ Updated in dc03859c718c5821be04fc7e169a0f71f663931f.